src/package/nuspec/Microsoft.TestPlatform.nuspec

```diff
@@ -8,7 +8,7 @@
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>The cross platform Microsoft Test Platform.</description>
-    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</licenseUrl>
+    <licenseUrl>https://www.visualstudio.com/microsoft-visual-studio-test-platform</licenseUrl>
     <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
```

Comment:
Good to have tags element `<tags>vstest visual-studio unittest testplatform mstest microsoft</tags>`. Feel free to add more tags

Reply:
👍